app/models/server/models/FederationPeers.js

```diff
@@ -1,7 +1,5 @@
 import { Meteor } from 'meteor/meteor';
-
 import { Base } from './_Base';
```

Comment:
I think this wlil solve the lint issue
```suggestion

import { Base } from './_Base';
```